about.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/60f96566f6.js" crossorigin="anonymous"></script>
+    <title>About</title>
+</head>
+<body>
+    <nav>
```

Comment:
add another div here and in the other page for the top navbar
![right](https://user-images.githubusercontent.com/58376020/124034680-d1836500-d9fb-11eb-9268-d2bfe65a5fb7.png)

Reply:
Issues resolved 👍